client/src/components/AlumniProfile.js

```diff
@@ -4,22 +4,19 @@ import LinkedInUpdate from "./LinkedInUpdate";
 
 /*
 props:
-- imageURL: string
-- name: string
-- college: string
-- location: string
-- company: string
-- jobTitle: string
+- details: Object containing:
+    - imageURL: string
+    - name: string
+    - college: string
+    - location: string
+    - company: string
+    - jobTitle: string
+    - email: string
 - isViewOnly: bool
 */
-export default class Profile extends Component {
+export default class AlumniProfile extends Component {
     render(){
-        const imageURL = this.props.imageURL;
-        const name = this.props.name;
-        const college = this.props.college;
-        const location = this.props.location;
-        const company = this.props.company;
-        const jobTitle = this.props.jobTitle;
+        const details = this.props.details;
         const isViewOnly = this.props.isViewOnly;
 
         const linkedInUpdate = (
```

Comment:
Let's add the `email` as prop to `LinkedInUpdate` here and for the StudentProfile!